packit/cli/init_local_yaml.py

```diff
@@ -0,0 +1,65 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+from pathlib import Path
+
+import click
+
+from packit.config import (
+    get_context_settings,
+    pass_config,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    "init-local-config",
+    context_settings=get_context_settings(),
+    short_help="Creates a localyaml config file for and ensures it's not version controlled",
+)
+@pass_config
+def init_local_yaml(config):
+    """Initialize packit-local yaml configuration file."""
+    config_path = Path(".packitLocal.yaml")
```

Comment:
I'm not sure how I feel about the config name, but I guess that can be easily adjusted.

Also I don't have a good suggestion for naming it, not to mention how convoluted this is becoming.